develop/reference.rst

```diff
@@ -1,3 +1,5 @@
+.. _sql_reference:
```

Comment:
This section is so high-level, I'm not sure why we want to link here. Is there a more specific place you had in mind?

Reply:
Then I think giving links to those two sections should be sufficient:
http://docs.citusdata.com/en/v9.5/develop/reference_workarounds.html & http://docs.citusdata.com/en/v9.5/develop/reference_ddl.html#modifying-tables